langstream-agents/langstream-vector-agents/src/main/java/ai/langstream/agents/vector/cassandra/CassandraAssetsManagerProvider.java

```diff
@@ -147,6 +152,11 @@ public void deployAsset() throws Exception {
                 }
             }
         }
+
+        @Override
+        public void deleteAsset() throws Exception {
```

Comment:
we can add a "default" method implementation here

Reply:
this code is not actually called, I'll follow up in the next pr